<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPassword</class>
 <widget class="QDialog" name="ForgotPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 #895061, stop:1 rgba(108, 85, 152, 186));;border-radius: 300px;

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border:0; padding:0; margin:0; background:0;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> border:0; border-radius:10;font:bold 20px;
background-color:#59253A;</string>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color:#FFF; font: bold 20px &quot;Sherif&quot;; padding:0; margin:0; </string>
        </property>
        <property name="text">
         <string> Forgot
 Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> font: bold 100px &quot;Sherif&quot;; color:#2F455C;
image:url(:/icon/icon/MITRA (1).png);
background:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>280</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>280</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px ;
margin-top: 1ex;
font: 16pt bold &quot;Sherif&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 #59253A, stop:1 #2D4159);
color:rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="username1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background:0; color:white; font:  &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Username</string>
            </property>
            <property name="buddy">
             <cstring>username3</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="username3">
            <property name="styleSheet">
             <string notr="true">background-color:white; color:#2f455c; border-radius:8px; padding:2;
font: bold &quot;Sherif&quot;;padding:7</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="a">
            <property name="styleSheet">
             <string notr="true">background:0; color:white; font:  &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Date of Birth</string>
            </property>
            <property name="buddy">
             <cstring>username3</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="DOB">
            <property name="styleSheet">
             <string notr="true">background-color:white; color:#2f455c; border-radius:8px; padding:2;
font: bold &quot;Sherif&quot;;padding:7</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="nickname">
            <property name="styleSheet">
             <string notr="true">background:0; color:white; font:  &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Your first nickname</string>
            </property>
            <property name="buddy">
             <cstring>username3</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Nickname">
            <property name="styleSheet">
             <string notr="true">background-color:white; color:#2f455c; border-radius:8px; padding:2;
font: bold &quot;Sherif&quot;;padding:7</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string>border:0; font: bold &quot;Sherif&quot;;
background-color:none;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true">border-radius:5px;
font: bold  &quot;Sherif&quot;;
background-color:#e68736;
color:rgb(255, 255, 255);
padding:9;</string>
           </property>
           <property name="text">
            <string>Change Password</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
